File: compiled_lang/cs/src/f41.loop.cs
// testbox: title="for (;;) with continue"
using System;

class F41Loop
{
    static void Main()
    {
        for (;;)
        {
            Console.Write("Enter your name (quit to exit): ");
            string answer = Console.ReadLine();

            if (string.IsNullOrWhiteSpace(answer))
                continue;

            if (answer == "quit")
                break;

            Console.WriteLine($"Hello {answer}!");
        }
    }
}

File: compiled_lang/go/f41.loop.go
// testbox: title="for ;; {} with continue"

package main

import (
	"bufio"
	"fmt"
	"os"
	"strings"
)

func main() {
	stdin := bufio.NewScanner(os.Stdin)

	for ; ; {
		fmt.Print("Enter your name (quit to exit): ")
		stdin.Scan()
		answer := stdin.Text()

		if strings.TrimSpace(answer) == "" {
			continue
		}

		if answer == "quit" {
			break
		}

		fmt.Println("Hello " + answer + "!")
	}
}
